docs/install/install-troubleshooting.md

```diff
@@ -79,6 +77,33 @@ If you don't see all containers running, then stop the running containers with `
 
 ---
 
+## Port Conflicts
+If you encounter an error like this:
+
+`Error starting userland proxy: Bind for 0.0.0.0:xxxx failed: port is already allocated`
+
+then ISLE may have encountered a conflict with the `xxxx` port identified in the error message.
+
+In MacOS, this can frequently be caused by a local Apache or Nginx web server, or local MySQL server.  
+
+You may need to remove or disable these local web servers before you can successfully install ISLE.
+
+* If you have a local Apache web server that ships with most MacOS machines may conflict with port 80, and can usually be disabled from a terminal using these commands:
+
+    * `sudo apachectl stop`
+    * `sudo launchctl unload -w /System/Library/LaunchDaemons/org.apache.httpd.plist 2>/dev/null`
+
+* If you have a local `Nginx` web server it may conflict with port 8080, and it can usually be disabled from a terminal using:
+
+    * `sudo nginx -s stop`
+
+Once your web server(s) have been disabled, resume the ISLE install process by repeating your last installation command, presumably `docker-compose up -d`.
+
+
+**Return to [Demo ISLE Installation](../install/install-demo.md).**
```

Comment:
I think we should be consistent about these pointers - there is a list at the bottom of the document and then this solo one here.  Not sure how to best do this, but perhaps we could revisit how this page is setup.

Reply:
I've edited this PR to resolve this comment. Thank you!